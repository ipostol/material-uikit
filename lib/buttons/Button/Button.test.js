'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _Button = require('./Button');

var _Button2 = _interopRequireDefault(_Button);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('Button component', function () {

  it('should render correctly', function () {

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(
      _Button2.default,
      null,
      'Button'
    ));

    expect(wrapper).toMatchSnapshot();
  });
});