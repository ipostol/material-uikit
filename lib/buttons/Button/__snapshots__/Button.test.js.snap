// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should render correctly 1`] = `
<button
  className="general secondary undefined small"
  onClick={[Function]}
  style={Object {}}
  type="button"
>
  <div
    className="alignLabel"
  >
    Button
  </div>
  <e
    background={true}
    duration={1000}
    opacity={0.25}
    radius={150}
    recenter={true}
  />
</button>
`;
